content/graphql/subscriptions.md

```diff
@@ -1,12 +1,25 @@
 ### Subscriptions
 
-In addition to fetching data using queries and modifying data using mutations, the GraphQL spec supports a third operation type, called `subscription`. GraphQL subscriptions are a way to push data from the server to the clients that choose to listen to real time messages from the server. Subscriptions are similar to queries in that they specify a set of fields to be delivered to the client, but instead of immediately returning a single answer, a result is sent every time a particular event happens on the server.
+In addition to fetching data using queries and modifying data using mutations, the GraphQL spec supports a third operation type, called `subscription`. GraphQL subscriptions are a way to push data from the server to the clients that choose to listen to real time messages from the server. Subscriptions are similar to queries in that they specify a set of fields to be delivered to the client, but instead of immediately returning a single answer, a channel is opened and a result is sent to the client every time a particular event happens on the server.
 
 A common use case for subscriptions is notifying the client side about particular events, for example the creation of a new object, updated fields and so on (read more [here](https://www.apollographql.com/docs/react/data/subscriptions)).
 
+#### Enable subscriptions
+
+To enable subscriptions, set the `installSubscriptionHandlers` property to `true`.
+
+```typescript
+GraphQLModule.forRoot({
+  typePaths: ['./**/*.graphql'],
```

Comment:
`typePaths` is not needed with the code first approach so I'm guessing we should remove it

Reply:
✔️